Tests/Marketplace/zip_packs.py

```diff
@@ -114,63 +109,80 @@ def remove_test_playbooks_from_signatures(path, filenames):
         logging.warning(f'Could not find signatures in the pack {os.path.basename(os.path.dirname(path))}')
 
 
-def download_packs_from_gcp(storage_bucket, gcp_path, destination_path, circle_build, branch_name):
+def get_pack_files(entry_path):
```

Comment:
It's hard for me to imagine that we don't have this functionality implemented elsewhere that you can import.

Reply:
Ok, where should I look for it or how is it called?